flow/scripts/variables.mk

```diff
@@ -18,11 +18,10 @@ export DESIGN_NICKNAME?=$(DESIGN_NAME)
 # Setup variables to point to other location for the following sub directory
 # - designs - default is under current directory
 # - platforms - default is under current directory
-# - work home - default is current directory
 # - utils, scripts, test - default is under current directory
 export DESIGN_HOME   ?= $(FLOW_HOME)/designs
 export PLATFORM_HOME ?= $(FLOW_HOME)/platforms
-export WORK_HOME     ?= .
+# WORK_HOME is set up in flow/Makefile
```

Comment:
Nit: I'm removing this comment in a followup PR. This is a comment left in place for a deleted line. Several variables are defined in flow/Makefile, WORK_HOME is not special in that regard.